models/public/face-detection-retail-0044/description/face-detection-retail-0044.md

```diff
@@ -0,0 +1,84 @@
+# face-detection-retail-0044
```

Comment:
Please, delete this file. Layout for public model is `public/<model name>/<model name>.md`